src/core/stdc/stdio.d

```diff
@@ -800,6 +840,129 @@ version (MinGW)
 }
+else version (LDC_MSVCRT14) // requires MSVCRT >= 14 (VS 2015)
+{
+  // MS define the *printf and *scanf function families inline, forwarding to 4
+  // basic functions.
+  // We implement corresponding wrappers around these basic functions and use
+  // special mangled names (__ldc_ prefix) to avoid duplicate symbols when
+  // linking against other C/C++ objects with their own versions.
+
+  private
+  {
+    enum _CRT_INTERNAL_PRINTF_STANDARD_SNPRINTF_BEHAVIOR = 2;
+
+    int __stdio_common_vfprintf(ulong _Options, FILE* _Stream,
+        in char* _Format, void* _Locale, va_list _ArgList);
+    int __stdio_common_vsprintf(ulong _Options, char* _Buffer, size_t _BufferCount,
+        in char* _Format, void* _Locale, va_list _ArgList);
+
+    int __stdio_common_vfscanf(ulong _Options, FILE* _Stream,
+        in char* _Format, void* _Locale, va_list _Arglist);
+    int __stdio_common_vsscanf(ulong _Options, in char* _Buffer, size_t _BufferCount,
+        in char* _Format, void* _Locale, va_list _ArgList);
+  }
+
+    // *printf:
+
+    ///
+    pragma(mangle, "__ldc_vfprintf")
+    int vfprintf(FILE* stream, in char* format, va_list args)
+    {
+        return __stdio_common_vfprintf(0, stream, format, null, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_fprintf")
+    int fprintf(FILE* stream, in char* format, ...)
+    {
+        va_list args;
+        va_start(args, format);
+        return vfprintf(stream, format, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_vprintf")
+    int vprintf(in char* format, va_list args)
+    {
+        return vfprintf(stdout, format, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_printf")
+    int printf(in char* format, ...)
+    {
+        va_list args;
+        va_start(args, format);
+        return vprintf(format, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_vsprintf")
+    int vsprintf(char* s, in char* format, va_list args)
+    {
+        return vsnprintf(s, size_t.max, format, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_sprintf")
+    int sprintf(char* s, in char* format, ...)
+    {
+        va_list args;
+        va_start(args, format);
+        return vsprintf(s, format, args);
+    }
+
+    // *scanf:
+
+    ///
+    pragma(mangle, "__ldc_vfscanf")
+    int vfscanf(FILE* stream, in char* format, va_list args)
+    {
+        return __stdio_common_vfscanf(0, stream, format, null, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_fscanf")
+    int fscanf(FILE* stream, in char* format, ...)
+    {
+        va_list args;
+        va_start(args, format);
+        return vfscanf(stream, format, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_vscanf")
+    int vscanf(in char* format, va_list args)
+    {
+        return vfscanf(stdin, format, args);
+    }
+
+    ///
+    pragma(mangle, "__ldc_scanf")
+    int scanf(in char* format, ...)
+    {
+        va_list args;
+        va_start(args, format);
+        return vscanf(format, args);
+    }
```

Comment:
Yes, i can. I've built ldc with msvc 2015. Only thing i've changed is new way fo acquiring std(in|out|err) with __acrt_iob_func and removing _f(put|get)c_nolock because they're already implemented in new msvc runtime.